modules/metering-install-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-prerequisites_{context}"]
+= Prerequisites
+
+Operator Metering requires the following components:
+
+* A StorageClass for dynamic volume provisioning. Metering supports a number of different storage solutions. See xref:../metering/metering-configuring-metering.adoc#configuring-storage[configuring storage for metering] for more details.
+* A Prometheus installation within the cluster configured to do Kubernetes cluster-monitoring. Openshift 4.x clusters have Prometheus enabled by default on installation.
```

Comment:
* Openshift -> {product-title}, and same thing about 4.x.
* maybe just say "configured for cluster monitoring". Not sure if you need to say Kubernetes?